build_tools/github_actions/fetch_job_status.py

```diff
@@ -0,0 +1,48 @@
+"""
+This script determines the job status for different jobs run
+as part of GitHub workflow based on RUN_ID and ATTEMPT
+
+Required environment variables:
+  - RUN_ID
+  - ATTEMPT
+"""
+
+from configure_ci import set_github_output
```

Comment:
We should pull `set_github_output` into a utils file instead of sharing across files this way.